lib/node_modules/@stdlib/namespace/alias2pkg/data/data.csv

```diff
@@ -68,30 +70,96 @@ base.abs2,"@stdlib/math/base/special/abs2"
 base.abs2f,"@stdlib/math/base/special/abs2f"
 base.absdiff,"@stdlib/math/base/utils/absolute-difference"
 base.absf,"@stdlib/math/base/special/absf"
+base.acartesianPower,"@stdlib/array/base/cartesian-power"
+base.acartesianProduct,"@stdlib/array/base/cartesian-product"
+base.acartesianSquare,"@stdlib/array/base/cartesian-square"
 base.acos,"@stdlib/math/base/special/acos"
+base.acosd,"@stdlib/math/base/special/acosd"
+base.acosf,"@stdlib/math/base/special/acosf"
 base.acosh,"@stdlib/math/base/special/acosh"
 base.acot,"@stdlib/math/base/special/acot"
+base.acotd,"@stdlib/math/base/special/acotd"
+base.acotf,"@stdlib/math/base/special/acotf"
 base.acoth,"@stdlib/math/base/special/acoth"
 base.acovercos,"@stdlib/math/base/special/acovercos"
 base.acoversin,"@stdlib/math/base/special/acoversin"
 base.acsc,"@stdlib/math/base/special/acsc"
+base.acscd,"@stdlib/math/base/special/acscd"
+base.acscdf,"@stdlib/math/base/special/acscdf"
+base.acscf,"@stdlib/math/base/special/acscf"
 base.acsch,"@stdlib/math/base/special/acsch"
 base.add,"@stdlib/math/base/ops/add"
 base.add3,"@stdlib/math/base/ops/add3"
 base.add4,"@stdlib/math/base/ops/add4"
 base.add5,"@stdlib/math/base/ops/add5"
 base.addf,"@stdlib/math/base/ops/addf"
+base.afilled,"@stdlib/array/base/filled"
```

Comment:
@Planeshifter You decided to export the "base" array packages. What is the reason for doing so?